### ISSUE TITLE
A way of 

### ISSUE THREAD

**Issue by shrektan** [RinteRface/bs4Dash#252]
Hi, 

I'm using bs4Dash internally. The network of the deploying environment is unable to access external URL. However, bs4Dash uses google fonts and ionicons, which require internet connections.

Is there a way of providing a local copy of those files? Currently, I have to modify the dependencies manually and I think it's not appropriate.

https://github.com/RinteRface/bs4Dash/blob/d3cda0d06ee19ef967e54c21743af943c8587666/R/deps.R#L13-L14

**Comment by b-rodrigues** [comment]
Hello

I’m in the same boat. For security reasons, I need to provide a local copy of the fonts, but this doesn’t seem to work. I’ve downloaded the fonts and try to include them in the dashboard header using:

```
tags$head(tags$link(rel = "stylesheet", type = "text/css", href = "css/source-sans-pro.css"))
```

but this doesn’t work, fonts still get used from google fonts. Any way to deactivate this would be greatly appreciated.

**Comment by DivadNojnarg** [comment]
I would do like here: https://google-webfonts-helper.herokuapp.com/fonts/source-sans-pro?subsets=latin but have to check whether all options can be found and match what is passed in https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700

**Comment by b-rodrigues** [comment]
So would including all fonts locally in the shiny app prevent it from fetching the fonts ? I’m not sure I get it.

**Comment by DivadNojnarg** [comment]
Does it work if you just do `shiny::suppressDependencies("googlefonts")` ?:

```r
library(shiny)
library(bs4Dash)

shinyApp(
  ui = dashboardPage(
    dashboardHeader(),
    dashboardSidebar(),
    dashboardBody(
      shiny::suppressDependencies("googlefonts"),
      dashboardBadge("Badge 1", color = "danger"),
      actionButton(
        inputId = "badge", 
        label = "Hello", 
        icon = NULL, 
        width = NULL, 
        dashboardBadge(1, color = "primary")
      )
    )
  ),
  server = function(input, output) { }
)
```

**Comment by b-rodrigues** [comment]
Yep, it works! Man, I wasn’t aware of shiny::suppressDependencies, this is great! Now my IT dep can sleep tight knowing that the app is gdpr compliant :)

**Comment by DivadNojnarg** [comment]
On my side, I'll try to make googlefonts and ionicons local deps. I just hope the package size is below 5mb. Then once pushed you'll be able to remove the `shiny::suppressDependencies` hack.

**Comment by b-rodrigues** [comment]
Ok, so I will read your book now, I’ve been eyeing it for some time, now I have no excuses :)
